doc/contributing/building-node-with-ninja.md

```diff
@@ -22,14 +22,13 @@ ninja: Entering directory `out/Release`
 
 The bottom line will change while building, showing the progress as
 `[finished/total]` build steps. This is useful output that `make` does not
-produce and is one of the benefits of using Ninja. Also, Ninja will likely
-compile much faster than even `make -j4` (or
-`-j<number of processor threads on your machine>`). You can still pass the
-number of processes to run for [Ninja][] using the environment variable `JOBS`.
-This will be the equivalent to the `-j` parameter in the regular `make`:
+produce and is one of the benefits of using Ninja. When using Ninja, builds
+are always run in parallel, based by default on the number of CPUs your
+system has. You can use the `-j` parameter to override this behavior,
+which is equivalent to the `-j` parameter in the regular `make`:
 
 ```bash
-JOBS=12 make
+make -j [number]
```

Comment:
I've never seen use of a space between `-j` and the number in regular `make` 
```suggestion
make -j[number]
```
Maybe t's clearer with a specific example
```suggestion
make -j4 # With this flag, Ninja will limit itself to 4 parallel jobs,
         # regardless of the number of cores on the current machine.
```

Reply:
LGTM